Makefile

```diff
@@ -2068,7 +2068,7 @@ mariadb_chainsaw_run: ## runs chainsaw tests for the mariadb operator, assumes t
 mariadb_chainsaw: export NAMESPACE = ${MARIADB_CHAINSAW_NAMESPACE}
 # Set the value of $MARIADB_CHAINSAW_NAMESPACE if you want to run the keystone
 # kuttl tests in a namespace different than the default (mariadb-chainsaw-tests)
-mariadb_chainsaw: input deploy_cleanup mariadb mariadb_deploy_prep ## runs chainsaw tests for the mariadb operator. Installs mariadb operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
+mariadb_chainsaw: input deploy_cleanup infra mariadb mariadb_deploy_prep ## runs chainsaw tests for the mariadb operator. Installs mariadb operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
```

Comment:
Actually, one thing I might suggest adding as well would be `make infra_cleanup` as a last step on L2077